### ISSUE TITLE
suggestion: support pipe(streams[]) api

### ISSUE THREAD

**Issue by okv** [juliangruber/multipipe#23]
Hi there.

It would be great if multipipe will accept streams no only like arguments (pipe(stream, ...)) but also like one array argument. Such alternative syntax convenient for passing dynamic streams array that was created by condition e.g.:

```js

var streams = [transformFields()];

if (params.anotherTranform) {
  streams.push(anotherTranform());
}

return pipe(streams);
```

currently you have to do something like `return pipe.apply(pipe, streams);` which in not very pleasant.

**Comment by juliangruber** [comment]
landed in 0.3.0

**Comment by okv** [comment]
That's great, thanks
